Phonebook.Frontend/src/app/modules/table/dialogs/table-settings-dialog/table-settings.dialog.html

```diff
@@ -61,3 +62,6 @@ <h2 i18n="TableSettingsDialog|Title of displayed table columns@@TableSettingsDia
     </div>
   </div>
 </mat-dialog-content>
+<mat-dialog-actions style="justify-content: flex-end;">
```

Comment:
There is an API from angular material for that :) 
```suggestion
<mat-dialog-actions align="end">
```
https://material.angular.io/components/dialog/overview#dialog-content